### PR TITLE
20200928 gabrielng post

### PR DESCRIPTION
**Pull request by jax79sg**
Hi, Please cross-check for security and embarrassing mistakes. Thanks.

### CODE REVIEW

File: _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+This project has adapted it for pathological voice classification by treating each demographic (e.g male, dysphonia, 41-65) as speakers. The rationale for this was because based on current research, the different demographics should have distinct characteristics from one another that would enable RawNet to distinguish between the different demographics. For example, male individuals would have a lower pitch than female individuals while individuals with a voice disorder would have a higher jitter and shimmer than healthy individuals.
+
+### Verifying the performance of RawNet
+Before adapting RawNet for classifying pathological voices, the original research was verified by replicating the research. This was done by training the RawNet model on voxCeleb, a dataset of speech recordings from celebrities. From the visualisation of the embeddings produced from the evaluation recordings by RawNet after training, it can be seen that all the speakers are in their own distinct clusters. Thus, it can be concluded that the original research was indeed successful in its purpose.
```
Comment (levan92) [inline _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md:87]: might want to state that the visualisation/test was done on the validation set(aka not seen in training)? Shows that the trained model is able to generalise to previously unseen voices.

File: _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+## Datasets
+There are a number of open-source pathological voice datasets available. These datasets are the Saarbruecken Voice Database (SVD), VOICED database and SLI-Database. Other databases such as the Massachusetts Eye and Ear Infirmary Database and the TalkBank pathology databases are unfortunately not open-source. In this project, the data used is obtained from the SVD because the data comes in .wav files, making it easier to start experimenting with the data. In all, all samples from 6 different disorder labels were downloaded and used for this project. This data can be downloaded here: https://drive.google.com/file/d/1P6Pip3ZWzKfG74p0EuWwSguIsOyAUFD_/view?usp=sharing
```
Comment (levan92) [inline _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md:49]: might want to give the size of the training/validation dataset used, as well as the breakdown for each class? if there was a train-val-test split good to show the size of each set as well. Especially because it was later implied (in the Final Results section) that there was not enough data at the moment for training.

File: _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+### Verifying the performance of RawNet
+Before adapting RawNet for classifying pathological voices, the original research was verified by replicating the research. This was done by training the RawNet model on voxCeleb, a dataset of speech recordings from celebrities. From the visualisation of the embeddings produced from the evaluation recordings by RawNet after training, it can be seen that all the speakers are in their own distinct clusters. Thus, it can be concluded that the original research was indeed successful in its purpose.
+
+![](../assets/images/2020-08-29-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings/voxceleb_eval_embeddings.png)
+
+### Final Results
+The classification with the RawNet model was conducted with the same scenarios as the baseline results
```
Comment (levan92) [inline _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md:92]: it is not clear to me if fine-tune training of the model was done on the SVD dataset or was it just used for validation?

File: _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+3. 1 vs 1 classification of disorder classes: ~0.55 F1-score
+
+The 5 high data demographics in this result are the 5 demographics that had the highest number of single utterances after data processing of around 1000 utterances. For example, healthy_male_41-65 is a demographic with this number of utterances, allowing for more training data.
+
+The baseline results indicate that the model is doing little better than randomly guessing which disorder label should be given to an unknown utterance. This is consistent across multiple models like Random Forests, Support Vector Machines and XGBoost. However, the hyperparameters of the models were not tuned so doing this may improve the baseline results.
+
+## Deep-learned Features with RawNet
+To improve on the baseline results, a deep learning model called RawNet is used. The original purpose of RawNet was for speaker verification with raw audio waveforms as input. The picture below illustrates the input and output of the model as well as its architecture.
+
+![](../assets/images/2020-08-29-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings/rawnet_architecture.png)
```
Comment (levan92) [inline _posts/2020-09-28-Pathological-voice-classification-with-deep-learned-embeddings-from-voice-recordings.md:82]: might want to elaborate a bit on what is the classifier used here. Was it a fully connected layer?
